ledger-core/virtual/object/object.go

```diff
@@ -428,8 +435,63 @@ func (sm *SMObject) stepMigrate(ctx smachine.MigrationContext) smachine.StateUpd
 		return ctx.Error(fmt.Errorf("failed to publish state report"))
 	}
 
+	sdlCallSummarySync := ctx.Share(&sm.SummaryDone, 0)
+
+	if !ctx.Publish(callsummary.BuildSummarySyncKey(sm.Reference), sdlCallSummarySync) {
+		ctx.Log().Warn(struct {
+			*log.Msg  `txt:"failed to publish call summary sync key"`
+			Reference string
+		}{
+			Reference: sm.Reference.String(),
+		})
+		return ctx.Error(fmt.Errorf("failed to publish call summary sync key"))
```

Comment:
What is the reason to use fmt.Errorf here? avoid it; use throw.New instead

Reply:
Fixed